CityPuzzle/Game Room/Join GameRoom/SeeEnteredRooms.xaml.cs

```diff
@@ -76,17 +78,28 @@ public static List<Room> GetData()
         void Sign_Click(object sender, EventArgs e)
         {
             string gamePin = GamePin.Text;
-            Navigation.PushAsync(new EntryGameRoomPage(gamePin));
-
+            Room current= EnteredRooms.SingleOrDefault(x => x.ID.Equals(gamePin));
+            if (current == null) Navigation.PushAsync(new EntryGameRoomPage(gamePin));
+            else RoomExistError(current);
         }
         async void Handle_ItemTapped(object sender, ItemTappedEventArgs e)
         {
-            if (e.Item == null)
+            if (e.Item != null)
             {
-                return;
+                SelectMsg(EnteredRooms[e.ItemIndex]);
             }
-    ((ListView)sender).SelectedItem = null;
+            MyListView.SelectedItem = null;
             Console.WriteLine(" " + e.Item);
         }
+        async void SelectMsg(Room selectedRoom)//exeotionus galima panaudoti
+        {
+            bool answer = await DisplayAlert("Demesio", "Ar norite testi zaidima- " + selectedRoom.ID, "Taip", "Ne");
+            if (answer == true) Console.WriteLine("Iveinu  {0}");
```

Comment:
This message box has no current functionality. Can users quit a game without it?

Reply:
You are right, but there is no questpage for gameRoom yet. How do you think the current questpage will fit or we need a new one?

Reply:
Use currennt questPage but read targets not from whole DB but from specific room quests list

Reply:
I will try to implemwnt this functionality next week

Reply:
Implemented in pull request #43